microcosm_pubsub/dispatcher.py

```diff
@@ -48,7 +48,7 @@ def handle_message(self, message):
         Handle a single message.
 
         """
-        media_type = message["media_type"]
+        media_type = message.get("media_type", "_")
```

Comment:
Are you OK with this part? Perhaps rather than explicitly using `_` we can add a parameter to the daemon for `microcosm_pubsub.default_key` and use that value so it is clearer what the key is for.